Adding members to Security Response Team (#232)
opensearch-project/.github: SECURITY.md

*Giving credit where credit is due, this policy is heavily influenced by the [Xen Project’s security response process](https://xenproject.org/developers/security-policy/), that was put to the test during the [embargo period for XSA-108 back in 2014](https://xenproject.org/2014/10/22/xen-project-security-policy-improvements-get-involved/) and improved its clarity around managing the pre-disclosure list and the deployment of fixes during embargo. We are standing on the shoulders of these battle-tested giants.*

> ## The Security Response Team (SRT)

Reviewer: ```suggestion
## Security Response Team (SRT)
```

> The OpenSearch Security Response Team (SRT) comprises of a subset of the project’s maintainers responsible for looking after the project’s security, including the security issue response process outlined below. New SRT members can only be nominated by current SRT members.

Reviewer: ```suggestion
The OpenSearch Security Response Team (SRT) is comprised of a subset of the project’s maintainers responsible for looking after the project’s security, including the security issue response process outlined below. New SRT members are nominated by current SRT members.
```
